build.sbt

```diff
@@ -1,24 +1,24 @@
 name := """tdr-transfer-frontend"""
 organization := "tna"
+maintainer := "tdr@nationalarchives.gov.uk"
 
 version := "1.0-SNAPSHOT"
 
 lazy val root = (project in file(".")).enablePlugins(PlayScala)
 
-scalaVersion := "2.12.8"
+scalaVersion := "2.13.0"
 
 libraryDependencies += guice
 libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.3" % Test
 
-val playPac4jVersion = "8.0.1"
-val pac4jVersion = "3.7.0"
+val playPac4jVersion = "9.0.0-RC3"
+val pac4jVersion = "4.0.0-RC3"
```

Comment:
It's probably OK at this early stage of the project, but in general it would be good to use stable versions rather than release candidates. Unless RCs in Scala are generally quite stable?

Reply:
I don't know, I generally just go for the most recent version and I've only had a few problems with it so far.

Reply:
Yeah, in this case we'll be using the dependencies in development for long enough to be confident that they're stable, but I wouldn't upgrade a production system to an RC unless there was a really strong reason.